torch/optim/optimizer.py

```diff
@@ -35,6 +36,9 @@
 )
 from torch.utils.hooks import RemovableHandle
 
+_T = TypeVar("_T")
```

Comment:
There's a T for TypeVar("T") below -- what does the underscore mean here?

Reply:
Underscore means the name is private to the file. I could have used the 'T' version as well - but I didn't want to make any implied guarantees about the type name having any external meaning.